### ISSUE TITLE
bowtie2 2.3.1 hangs

### ISSUE THREAD

**Issue by tomas-pluskal** [BenLangmead/bowtie2#99]
Hi,

Recently I noticed our transcriptome assemblies using Trinity were getting stuck at the bowtie2 step. After a bit of debugging, I could nail it down to this command:
````
bowtie2 --local -k 2 --threads 20 -f --score-min G,46,0 -x inchworm.K25.L25.DS.fa.min100 both.fa
````
This command seems to complete fine with versions 2.2.5 or 2.3.0, but it hangs (never completes, no error reported) with version 2.3.1.

If you like, I can upload the data files somewhere for debugging (it's about 10 GB though).

**Comment by nylander** [comment]
If this is on (Ubuntu) Linux, does it complete with `--threads 1`? There is an issue with the current (Ubuntu) Linux libraries for running bowtie2 in parallel.

**Comment by tomas-pluskal** [comment]
Yes, it does finish fine when running in a single thread.

**Comment by ch4rr0** [comment]
I was able to recreate the issue. I am working on finding the cause of the problem. Will reply with results.

**Comment by ch4rr0** [comment]
This issue has been fixed in Bowtie 2 v2.3.2.

**Comment by wenqinchao** [comment]
I downloaded Bowtie2 v2.3.2, then trinity run successfully
